src/dotnet/commands/dotnet-build/BuilderCommandApp.cs

```diff
@@ -9,14 +9,17 @@ internal class BuilderCommandApp : CompilerCommandApp
     {
         public const string BuildProfileFlag = "--build-profile";
         public const string ForceUnsafeFlag = "--no-incremental";
+        public const string NoDependenciesFlag = "--no-dependencies";
 
         public bool BuildProfileValue => OptionHasValue(BuildProfileFlag);
         public bool ForceUnsafeValue => OptionHasValue(ForceUnsafeFlag);
+        public bool NoDependenciesValue => OptionHasValue(NoDependenciesFlag);
```

Comment:
It kinda breaks the naming convention I gave to the argument properties. On the other hand it makes expressions containing the property easier to read.

I'll go ahead and rename all of those 3 :)